files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```
@@ -195,6 +195,12 @@
     replacement: etcd
   - source_labels: [__address__]
     target_label: instance
+[[ if and (ne .ClusterType "management_cluster") (.WorkloadClusterETCDDomain) ]]
```

My OCD would be happier if this condition matched the one in `scrapeconfigs/resource.go`, i.e.:

```suggestion
[[ if and (eq .ClusterType "workload_cluster") (.WorkloadClusterETCDDomain) ]]
```